### DISCUSSION TITLE
CreateNotificationChannel in 11.1.0

### DISCUSSION THREAD

**Discussion by vallgrenerik** [thudugala/Plugin.LocalNotification#479] [Q&A] [answered]
Hi @thudugala 

I use your awesome plugin in a couple of net8.0-android & net8.0-iOS applications (previously known as Xamarin.Android & Xamarin.iOS/Xamarin.Native).

In the previous version (10.1.8) I could create a notification channel like this:
```
LocalNotificationCenter.CreateNotificationChannel(
	new NotificationChannelRequest()
	{
		Id = "MyID",	
		Name = "MyName",
		Description = "MyDescription",
	});

```

In the new (11.1.0) version, how do I create it there? 😊

Thanks in advance!

**Comment by thudugala** [comment]
@vallgrenerik 

https://github.com/thudugala/Plugin.LocalNotification/wiki/%5BAndroid---=-26%5D-Notification-Channel

**Reply by vallgrenerik** [reply]
@thudugala  
Yes thanks, but my application is a net8.0-android native application, not MAUI-one.

**Reply by thudugala** [reply]
@vallgrenerik hmm you make a good point. Let me make these methods public again

**Reply by vallgrenerik** [reply]
@thudugala Thanks!! 😊Looking forward to an update. I really appreciate your plugin.

**Comment by thudugala** [comment]
@vallgrenerik is your application Xamarin or .Net Manu

from version 11 onwards plugin only supports  .Net Manu

**Reply by vallgrenerik** [reply]
@thudugala neither, I'm using a simple `dotnet new android` and `dotnet new ios` . That's the one MAUI rely on. It worked awesome and as expected until you changed the visibility of the methods.

**Comment by thudugala** [comment, marked as answer]
@vallgrenerik try [11.1.1](https://www.nuget.org/packages/Plugin.LocalNotification/11.1.1)

**Reply by vallgrenerik** [reply]
Thank you @thudugala! It seems to work as expected!
